libraries/SITL/SIM_GPS.h

```diff
@@ -92,6 +96,13 @@ class GPS : public SerialDevice {
         double roll_deg;
         double pitch_deg;
         bool have_lock;
+        uint8_t num_satellites;
+
+        // Get heading [rad], where 0 = North in WGS-84 coordinate system
+        float heading() const WARN_IF_UNUSED;
```

Comment:
https://github.com/ArduPilot/ardupilot/pull/24525

These changes are now part of another PR, and will need to get merged first.